static/js/message_edit_history.js

```diff
@@ -11,6 +11,7 @@ import * as message_lists from "./message_lists";
 import {page_params} from "./page_params";
 import * as people from "./people";
 import * as popovers from "./popovers";
+import * as rm from "./rendered_markdown";
```

Comment:
We always import modules with their name rather than this sort of variable name in some files; can you do that here?  

@andersk is there an `eslint` rule we can turn on to enforce this?